session.go

```diff
@@ -168,3 +168,40 @@ func (s Session) CaptchaImage(iden string) (image.Image, error) {
 
 	return m, nil
 }
+
+// SubmissionInfo fetches the metadata for a string of comma-separated fullnames
+// (e.g,"t3_5pbezh,t3_5pbezh").  Uses https://www.reddit.com/dev/api/#GET_by_id_{names}
+func (s Session) SubmissionInfo(fullnameList string) ([]*Submission, error) {
```

Comment:
I'd rather the parameter have more structure by requiring a `[]string` and we join that inside the function.